stake-o-matic/stake-o-matic.sh

```diff
@@ -10,7 +10,8 @@ curl -sSf https://raw.githubusercontent.com/solana-labs/solana/v1.0.0/install/so
         --data-dir ./solana-install \
         --config ./solana-install/config.yml
 
-export PATH="$PWD/solana-install/releases/$solana_version/solana-release/bin/:$PATH"
+export PATH="$(cd "$PWD"/solana-install/releases/"$solana_version"*/solana-release/bin/;pwd):$PATH"
```

Comment:
`realpath "$PWD"/solana-install/releases/"$solana_version"*/solana-release/bin/` ?

Reply:
Sure, done

Reply:
let's have a fun shell talk; I like this new addition of `realpath` to coreutils; Recently, I got a habit of adding `-e` to `realpath` to ensure the last component actually exists; unless doing so, `realpath` doesn't ensure the existence of the last component:

```
$ realpath  ~/b1n
/home/ryoqun/b1n
$ realpath -e ~/b1n
realpath: /home/ryoqun/b1n: No such file or directory
```

So, it could introduce some hard-to-debug bugs in some cases.

Reply:
TIL :slightly_smiling_face: